packages/react-core/src/components/Wizard/Wizard.tsx

```diff
@@ -429,12 +429,17 @@ export class Wizard extends React.Component<WizardProps, WizardState> {
       activeStep
     };
 
+    const divStyles = {
+      ...(height ? { height } : {}),
+      ...(width ? { width } : {})
+    };
+
     const wizard = (
       <WizardContextProvider value={context}>
         <div
           {...rest}
           className={css(styles.wizard, activeStep && activeStep.isFinishedStep && 'pf-m-finished', className)}
-          {...(height && { style: { height } })}
+          style={Object.keys(divStyles).length ? divStyles : undefined}
```

Comment:
We have CSS vars you can use here or via `setProperty()` to control the width and max-width. That's usually preferred as we may use that variable to help calculate some other property value that references the modal's width.

`--pf-c-modal-box--Width` - sets the width
`--pf-c-modal-box--MaxWidth` - sets the max-width

Helpful if you want the modal to be say, 50%, but not exceed 800px (`--pf-c-modal-box--Width: 50%; --pf-c-modal-box--MaxWidth: 800px;`)

Reply:
does it apply on the wizard when it's not displayed inside a modal?

Reply:
Oh sorry @boaz0, I read this wrong, I thought the width applied to the modal. My suggestion would not change anything with the wizard at all - it only change the modal. When the wizard is used in a modal, it is set to fill the full width and height of the modal. So by updating the modal's dimensions, the wizards dimensions automatically change to match the modal's.

If this feature is only to change the width of the wizard when in a modal, personally I think we should change the width of the modal. And instead of using the var I suggested, I just looked up the modal docs and `<Modal>` supports a `width` prop.

But if we also want the ability for someone to pass a width to the wizard when used outside of a modal, the approach in the PR makes sense. Another approach we could take for that, assuming the wizard is used in a page section, is to allow a custom `max-width` on page sections, and allow the user to set the max width on the page section the wizard is in.